Resources/Maps/_NF/Shuttles/akupara.yml

```diff
@@ -0,0 +1,4264 @@
+      pos: 6.5,-1.5
+      parent: 1
+  - uid: 374
+    components:
+    - type: Transform
+      rot: -1.5707963267948966 rad
+      pos: 6.5,2.5
+      parent: 1
+- proto: PoweredSmallLight
+  entities:
+  - uid: 375
+    components:
+    - type: Transform
+      rot: 1.5707963267948966 rad
+      pos: 3.5,-12.5
+      parent: 1
+  - uid: 376
+    components:
+    - type: Transform
+      rot: 1.5707963267948966 rad
+      pos: 3.5,-8.5
+      parent: 1
+- proto: PuddleVomit
+  entities:
+  - uid: 377
+    components:
+    - type: Transform
+      pos: 5.5,8.5
+      parent: 1
+- proto: Rack
+  entities:
+  - uid: 378
+    components:
+    - type: Transform
+      pos: -0.5,2.5
+      parent: 1
+  - uid: 379
+    components:
+    - type: Transform
+      pos: 5.5,3.5
+      parent: 1
+  - uid: 380
+    components:
+    - type: Transform
+      pos: 2.5,-10.5
+      parent: 1
+- proto: Railing
+  entities:
+  - uid: 381
+    components:
+    - type: Transform
+      rot: 1.5707963267948966 rad
+      pos: -5.5,0.5
+      parent: 1
+  - uid: 382
+    components:
+    - type: Transform
+      rot: -1.5707963267948966 rad
+      pos: -3.5,0.5
+      parent: 1
+  - uid: 383
+    components:
+    - type: Transform
+      rot: 3.141592653589793 rad
+      pos: -4.5,-0.5
+      parent: 1
+  - uid: 384
+    components:
+    - type: Transform
+      pos: -4.5,1.5
+      parent: 1
+- proto: RailingCornerSmall
+  entities:
+  - uid: 385
+    components:
+    - type: Transform
+      rot: 1.5707963267948966 rad
+      pos: -3.5,1.5
+      parent: 1
+  - uid: 386
+    components:
+    - type: Transform
+      rot: 3.141592653589793 rad
+      pos: -5.5,1.5
+      parent: 1
+  - uid: 387
+    components:
+    - type: Transform
+      rot: -1.5707963267948966 rad
+      pos: -5.5,-0.5
+      parent: 1
+  - uid: 388
+    components:
+    - type: Transform
+      pos: -3.5,-0.5
+      parent: 1
+- proto: RandomPosterLegit
+  entities:
+  - uid: 389
+    components:
+    - type: Transform
+      rot: 3.141592653589793 rad
+      pos: 2.5,-7.5
+      parent: 1
+  - uid: 390
+    components:
+    - type: Transform
+      rot: 3.141592653589793 rad
+      pos: 2.5,-6.5
+      parent: 1
+- proto: RandomProduce
+  entities:
+  - uid: 391
+    components:
+    - type: Transform
+      pos: 0.5,7.5
+      parent: 1
+- proto: RandomSpawner
+  entities:
+  - uid: 392
+    components:
+    - type: Transform
+      pos: 3.5,-12.5
+      parent: 1
+  - uid: 393
+    components:
+    - type: Transform
+      pos: 4.5,13.5
+      parent: 1
+  - uid: 394
+    components:
+    - type: Transform
+      rot: 1.5707963267948966 rad
+      pos: -0.5,-4.5
+      parent: 1
+- proto: RandomSpawner100
+  entities:
+  - uid: 395
+    components:
+    - type: Transform
+      rot: 1.5707963267948966 rad
+      pos: -3.5,5.5
+      parent: 1
+  - uid: 396
+    components:
+    - type: Transform
+      pos: 4.5,0.5
+      parent: 1
+- proto: RandomVendingDrinks
```

Comment:
Remove

Reply:
Would a non-random drink vendor (IE Starkist vendor) be acceptable? It's fine if not, it's mainly fluff just to tie the park bench theme.

Reply:
No food / drink vends

Reply:
Adjustment made according to guideline clarifications